apps/src/code-studio/components/ShareAllowedDialog.jsx

```diff
@@ -297,27 +293,26 @@ class ShareAllowedDialog extends React.Component {
                     <img style={styles.thumbnailImg} src={thumbnailUrl} />
                   </div>
                   <div>
-                    <p style={{fontSize: 20}}>{i18n.shareCopyLink()}</p>
-                    <input
-                      type="text"
-                      id="sharing-input"
-                      onClick={select}
-                      readOnly
-                      value={this.props.shareUrl}
-                      style={{cursor: 'copy', width: 450}}
-                    />
                     <button
                       type="button"
-                      id="share-dialog-copy-button"
+                      id="sharing-input-copy-button"
                       style={{
                         ...styles.button,
                         ...styles.copyButton,
                         ...(this.state.hasBeenCopied && styles.copyButtonLight)
                       }}
                       onClick={wrapShareClick(this.copy, 'copy')}
+                      value={this.props.shareUrl}
```

Comment:
I needed a value in order to get the UI tests to pass. The better option here would be to click this button then access the link from the [clipboard](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard) but that doesn't work in Firefox and doesn't work locally with out localhost domain setup (it requires a secure context which localhost:3000 would be allow listed for but localhost-studio.code.org:3000 would not be).